modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= Support for {oadp-full}
+
+.Supported versions of {oadp-short}
+[width="100%",cols="10%,12%,12%,13%,13%,20%,20%,options="header"]
+|===
+
+|Version
+|{OCP-short} version
```

Comment:
Same comment as in the other PR, I think it would be better to replace this attribute throughout the table with just the term, since it doesn't render on enterprise docs:
```suggestion
|OCP version
```